composables/getters/facetGetters.ts

```diff
@@ -0,0 +1,34 @@
+import { buildBreadcrumbs } from "../helpers/buildBreadcrumbs"
+import { Breadcrumb } from "~~/pages/types"
+import { Maybe, PrCategory } from "~~/server/types/GraphQL"
+
+const getCategoryTree = (searchData: { categories: Maybe<Array<Maybe<PrCategory>>> }) => {
+  if (!searchData) {
```

Comment:
Make it onliner if (!searchData) return []

Reply:
Resolved